frontend/src/components/LandingPage/Tools/Tools.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Jumbotron, Media } from 'reactstrap';
+import ReactPicture from './Pictures/ReactPicture.png'
+import NodePicture from './Pictures/NodePicture.png'
+import JiraPicture from './Pictures/JiraPicture.png'
+import GithubPic from './Pictures/GithubPic.png'
+
+const Tools = (props) => {
```

Comment:
Can you change this to a functional component please.

Reply:
Changed it to a functional component.